pkg/webui/components/file-input/index.js

```diff
@@ -48,13 +49,15 @@ export default class FileInput extends Component {
     id: PropTypes.string.isRequired,
     image: PropTypes.bool,
     imageClassName: PropTypes.string,
+    largeFileWarningMessage: PropTypes.string.isRequired,
     maxSize: PropTypes.number,
     mayRemove: PropTypes.bool,
     message: PropTypes.message,
     name: PropTypes.string.isRequired,
     onChange: PropTypes.func.isRequired,
     providedMessage: PropTypes.message,
     value: PropTypes.oneOfType([PropTypes.string, PropTypes.shape({})]),
+    warningSize: PropTypes.number.isRequired,
```

Comment:
```suggestion
    warningSize: PropTypes.number.isRequired,
```

It should not be required. It's perfectly fine to use the `<FileInput />` without any warning popping up for large file sizes.